addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1267,6 +1295,18 @@ export function isFontSize(node, props) {
     const element = closestElement(node);
     return getComputedStyle(element)['font-size'] === props.size;
 }
+/**
+ * Return true if the given node classlist contains `props.className`.
+ *
+ * @param {Object} props
+ * @param {Node} node A node to compare the font-size against.
+ * @param {String} props.className The name of the class.
+ * @returns {boolean}
+ */
+export function isClassName(node, props) {
```

Comment:
I can't reconcile what the function does with it its name, even with the doc. Shouldn't it be something like `hasClass(node, className)` ? And at that point, it's basically `closestElement(node).classList.contains(className)`, I don't see why we need an util just for that as that is very basic element handling. Moreover, this is only used once so I sugget it be inlined at call site.